app/lib/app/SecvisogramPage/View/FormEditor/schema.js

```diff
@@ -4232,7 +4232,7 @@ export default /** @type {const} */ ({
                     },
                     itemName: {
                       itemNameTranslationKey: 'arrays.productGroupsItemName',
-                      itemNameField: 'group_id',
+                      itemNameField: 'summary',
```

Comment:
As discussed: This change should be reverted.
```suggestion
                      itemNameField: 'group_id',